### DISCUSSION TITLE
Announcing to a Gitter Community Room

### DISCUSSION THREAD

**Discussion by sdelamo** [jreleaser/jreleaser#877] [Ideas]
Is it possible to announce to a Gitter Community Room? Would you be interested in receiving PRs to do that?

**Comment by aalmiray** [comment]
Gitter integration already exists, at the moment it happens via incoming webhooks the notification is sent to the sidebar instead of the main message pane; for that a bot is required IIRC which would also need a registered app and access grants. 

This being said, contributions are welcome anytime 😄
